src/DynamoCoreWpf/ViewModels/Core/AnnotationViewModel.cs

```diff
@@ -614,6 +614,26 @@ private bool CanChangeFontSize(object obj)
         {
             return true;
         }
+
+        private DelegateCommand toggleIsVisibleGroupCommand;
+
+        /// <summary>
+        /// Command to toggle this group's node preview visibility.
+        /// belongs to.
+        /// </summary>
+        [JsonIgnore]
+        public DelegateCommand ToggleIsVisibleGroupCommand
+        {
+            get
+            {
+                if (toggleIsVisibleGroupCommand == null)
```

Comment:
this should have braces {}

Reply:
addressed here: https://github.com/DynamoDS/Dynamo/pull/13729